tutorials/publish.md

```diff
@@ -62,13 +62,13 @@ _If you are not familiar with Docker you may want to take a look at its official
 
 Build your own tutorial Docker image directly from our git repo using the following command:
 
-```sh
+```bash
```

Comment:
Any reason for changing these simple shell commands to bash specific highlighting?

Reply:
The highlighting at the deployed web page is kind of ugly with `sh`.
Eg:
`docker build -t snet_publish_service ...`
The `-t` is red here.

Reply:
To keep the standard, I've changed it back to `sh` and updated some other parts of the repo too.
Tell me what do you think?